dist/ci/testenv-tumbleweed/Dockerfile

```diff
@@ -5,9 +5,9 @@ FROM opensuse/tumbleweed
 RUN zypper -n ar http://download.opensuse.org/repositories/openSUSE:/Tools/openSUSE_Factory/ openSUSE:Tools
 RUN zypper --gpg-auto-import-keys ref
 
-RUN zypper in -y osc python3-nose python3-httpretty python3-pyxdg python3-PyYAML \
+RUN zypper in -y osc python3-pytest python3-httpretty python3-pyxdg python3-PyYAML \
```

Comment:
you better create another PR to add both. You can still remove nose in this.

Reply:
well, it is still WIP. I plan to temporary just for testing install it here. Locally it failed after several tests for unavailable obs API, but it is probably caused by my local machine computation power.

Reply:
that's my point: you can't test pytest without it being in the OBS container

Reply:
well, I plan to install it with zypper for now as temporary solution for testing that all tests can pass.

Reply:
trick we use often in yast where we have shared base image for yast and if additional library is needed for one specific module we install it with zypper as part of run to not grow too much generic image.